<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>abuild</class>
 <widget class="QDialog" name="abuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listDirectories">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>391</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Symbols</string>
      </attribute>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listSymbols">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>391</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Warnings</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>351</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QRadioButton" name="radioButtonW">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>No Warnings (-W)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonWall">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>All Warnings (-Wall)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonWerror">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Warnings as errors (-Werror)</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Optimization</string>
      </attribute>
      <widget class="QComboBox" name="comboBoxOptimization">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>-O0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-O1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-O2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-O3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-Os</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>abuild</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>abuild</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
